README.Rmd

```diff
@@ -54,6 +54,7 @@ install.packages("IsoMemo")
 ```{r example}
 options(repos = c(getOption("repos"), PANDORA = "https://Pandora-IsoMemo.github.io/drat/"))
 install.packages("IsoMemo")
+devtools::load_all(".")
```

Comment:
Why do you require a load_all? This should not be the case.

Reply:
yes it has been deleted. but initially, this gave an error because the environment did not recongnize the package when I ran `library(IsoMemo)` because of the name change, so that's why I had to use `load_all()` , so that the README.rmd file can be knitted officially for the package.